doc-utils/CodeBlock/CodeBlock.scss

```diff
@@ -25,6 +25,7 @@
     width: 100%;
     padding: jkl.$spacing-s;
     max-width: min(#{jkl.rem(750px)}, 50vw);
+    @include jkl.use-font-family("Fremtind Grotesk Mono");
```

Comment:
Er det lov å si at jeg ikke synes mono-fonten vår funker så bra som kodefont? 😅 Hos meg virker det uansett som det faller tilbake til standard `monospace` i kodeeksemplene 🤔

Reply:
Aha! Sjekket ikke så grundig. Kan også la det være 😅 Har en langsiktig drøm om å få inn et Jøkul-tema for syntax highlighting, og da må jo fonten også være med! 😄

Reply:
> Er det lov å si at jeg ikke synes mono-fonten vår funker så bra som kodefont? 😅 

Skjønner ikke hva du mener 👀 

![Skjermbilde som viser innstillingene for Visual Studio Code. Fremtind Grotesk Mono er satt som font.](https://user-images.githubusercontent.com/1223410/194878048-102d60bf-ed60-4a62-9e7d-ed9f724eef29.png)